example/usps_api/config/config_parser.cc

```diff
@@ -1,25 +1,44 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
 #include "config_parser.h"
 #include "json/json.h"
 #include <iostream>
 #include <fstream>
 
-// reads configuration file or creates one if DNE
+// Reads the configuration file or creates one if not present.
 void Config::Initialize() {
-  char filename[] = "config/config.json";
+  const std::string filename = "config/config.json";
   std::ifstream file(filename, std::ifstream::binary);
   if (file.good()) {
     std::cout << "Reading from config.json" << std::endl;
     Json::Value root;
-    file >> root;
-    Config::ParseConfig(root);
+    Json::CharReaderBuilder builder;
+    std::string errs;
+    bool isValidJson = Json::parseFromStream(builder, file, &root, &errs);
```

Comment:
https://google.github.io/styleguide/cppguide.html#Variable_Names has guidance on the naming of variables.

Reply:
Done!